matplotcheck/base.py

```diff
@@ -216,6 +231,13 @@ def assert_caption_contains(self, strings_exp):
             list of lists in which each internal list is a list of strings.
             If ``strings_exp = None``, assertion passes.
             If ``strings_exp == []``, assertion passes.
+        message : string
+            The error message to be displayed if the caption does not contain one of the
+            expected strings. If `message` contains ``'{0}'``, it
+            will be replaced with the first expected string not found in the caption.
+        message_no_caption : string
+            The error message to be displayed if no caption exists in the appropriate
```

Comment:
The error message returned if the plot doesn't have a caption.

@ryla5068 does it test that the caption is in a particular location on the plot as well or just that it contains a caption somewhere?